bookkeeper-dist/src/assemble/bin-server.xml

```diff
@@ -45,6 +45,24 @@
         <include>${basedir}/*.txt</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>../src/main/resources/deps</directory>
+      <outputDirectory>/deps</outputDirectory>
+      <includes>
+        <include>javax.servlet-api-3.1.0/CDDL+GPL-1.1</include>
+        <include>netty-4.1.12.Final/LICENSE.base64.txt</include>
```

Comment:
nit: can this simplified with netty-4.1.12.Final/*?

Reply:
done